src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/main.py

```diff
@@ -256,14 +256,14 @@ def worker_thread(
 
     organizations = Organizations(
         role=boto3,
-        account_id=account_id
+        account_id=account_id,
+        cache=cache
     )
     ou_id = organizations.get_parent_info().get("ou_parent_id")
 
     account_path = organizations.build_account_path(
         ou_id,
-        [],  # Initial empty array to hold OU Path,
-        cache
+        [],  # Initial empty array to hold OU Path
```

Comment:
Could you also apply the same patch to the other usage of `build_account_path` on line 494 in this file?